website/docs/r/compute_subnetwork.html.markdown

```diff
@@ -1,98 +1,177 @@
+  except the last character, which cannot be a dash.
+* `network` -
+  (Required)
+  The network this subnet belongs to.
+  Only networks that are in the distributed mode can have subnetworks.
 
 
 - - -
 
-* `secondary_ip_range` - (Optional, [Beta](/docs/providers/google/index.html#beta-features)) An array of configurations for secondary IP ranges for VM instances contained in this subnetwork. Structure is documented below.
+* `description` -
+  (Optional)
+  An optional description of this resource. Provide this property when
+  you create the resource. This field can be set only at resource
+  creation time.
+* `enable_flow_logs` -
+  (Optional)
+  Whether to enable flow logging for this subnetwork.
+* `secondary_ip_range` -
+  (Optional)
+  An array of configurations for secondary IP ranges for VM instances
+  contained in this subnetwork. The primary IP of such VM must belong
+  to the primary ipCidrRange of the subnetwork. The alias IPs may belong
+  to either primary or secondary ranges.  Structure is documented below.
+* `private_ip_google_access` -
+  (Optional)
+  Whether the VMs in this subnet can access Google services without
+  assigned external IP addresses.
+* `region` -
+  (Optional)
+  URL of the region where the regional address resides.
+  This field is not applicable to global addresses.
+* `project` (Optional) The ID of the project in which the resource belongs.
+    If it is not provided, the provider project is used.
 
 The `secondary_ip_range` block supports:
-
-* `range_name` - (Required) The name associated with this subnetwork secondary range, used when adding an alias IP range to a VM instance.
-
-* `ip_cidr_range` - (Required) The range of IP addresses belonging to this subnetwork secondary range. Ranges must be unique and non-overlapping with all primary and secondary IP ranges within a network.
+* `range_name` -
+  (Required)
+  The name associated with this subnetwork secondary range, used
+  when adding an alias IP range to a VM instance. The name must
+  be 1-63 characters long, and comply with RFC1035. The name
+  must be unique within the subnetwork.
+* `ip_cidr_range` -
+  (Required)
+  The range of IP addresses belonging to this subnetwork secondary
+  range. Provide this property when you create the subnetwork.
+  Ranges must be unique and non-overlapping with all primary and
+  secondary IP ranges within a network. Only IPv4 is supported.
+    
 
 ## Attributes Reference
 
-In addition to the arguments listed above, the following computed attributes are
-exported:
-
-* `gateway_address` - The IP address of the gateway.
-
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `creation_timestamp` -
+  Creation timestamp in RFC3339 text format.
+* `gateway_address` -
+  The gateway address for default routes to reach destination addresses
+  outside this subnetwork.
+* `fingerprint` -
+  Fingerprint of this resource. A hash of the contents stored in this
```

Comment:
nit: this feels like a lot of information for something we're hoping the user ignores

Reply:
Agreed, substantially reduced.